specification/ai/Azure.AI.Client/vector_stores/file_batches/models.tsp

```diff
@@ -0,0 +1,45 @@
+namespace Azure.AI.Client.Agents;
```

Comment:
Same here... looks like all these should be ain a models.tsp file.

Reply:
Same story

Reply:
restructured